src/v2/Apps/Auction2/Components/Form/Utils/formatError.ts

```diff
@@ -0,0 +1,17 @@
+export const formatError = error => {
+  const errorMessages = (() => {
+    switch (true) {
+      case Array.isArray(error): {
+        return error.map(e => e.message)
+      }
+      case typeof error === "string": {
+        return [error]
+      }
+      case error?.message: {
+        return [error.message]
+      }
+    }
+  })()
+
+  return errorMessages
+}
```

Comment:
An amusing amount of code is required to deal with our wildly inconsistent API errors.